containers/images/foreman/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release \
+    							 http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm \
+    							 https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install qpid-tools && \
+    yum -y install foreman tfm-rubygem-puma \
+    							 tfm-rubygem-katello tfm-rubygem-foreman_ansible tfm-rubygem-foreman_discovery tfm-rubygem-foreman_hooks \
+    							 tfm-rubygem-foreman_openscap tfm-rubygem-foreman_remote_execution tfm-rubygem-foreman_templates && \
+    yum clean all
+
+COPY container-assets/settings.yaml /etc/foreman/settings.yaml
+COPY container-assets/katello.yaml /etc/foreman/plugins/katello.yaml
+
+# Generate apipie:cache:index
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install foreman-sqlite && \
+		yum clean all
+
+RUN DATABASE_URL=sqlite3::memory: foreman-rake db:create db:migrate apipie:cache:index
+
+RUN yum -y remove foreman-sqlite && \
```

Comment:
note this doesnt remove all dependent packages that get installed (e.g. tfm-ror51-rubygem-sqlite3) either add that as well or consider using something like yum history (https://github.com/theforeman/forklift/pull/703/files#diff-e73050ae9f2baca99ce3cc574adee190R27)